### DISCUSSION TITLE
Map background for BIIGLE

### DISCUSSION THREAD

**Discussion by BernardP437** [biigle/core#507] [Ideas]
Could the map in BIIGLE have a seabed background? We used the EMODNET mean depth layer in a recent project. 

https://www.emodnet-bathymetry.eu/data-products/which-information-layers

and

https://portal.emodnet-bathymetry.eu/?menu=19 

Bernard

**Comment by mzur** [comment]
No, sorry. BIIGLE used to have a feature where you could upload "overlays", i.e. single images that were displayed on the map, but this was [removed](https://github.com/biigle/geo/issues/17) in favor of the image and annotation location reports that can be imported in a GIS. Also, we don't plan to implement more GIS features in BIIGLE and rather focus on the annotation features. I suggest that you use a GIS and visualize the images/annotations (from a report) together with the appropriate map layers there.

Or do you have a use case where you acutally need this for annotation rather than just visualization?

**Reply by BernardP437** [reply]
Hi Martin,

That's fine, I understand.
